File: 09/CS/CustomControls/09CustomControls/Client/UserCode/SliderControl905.cs
using System;
using System.Linq;
using System.IO;
using System.IO.IsolatedStorage;
using System.Collections.Generic;
using Microsoft.LightSwitch;
using Microsoft.LightSwitch.Framework.Client;
using Microsoft.LightSwitch.Presentation;
using Microsoft.LightSwitch.Presentation.Extensions;

namespace LightSwitchApplication
{
    public partial class SliderControl905
    {
        partial void SliderControl905_InitializeDataWorkspace(global::System.Collections.Generic.List<global::Microsoft.LightSwitch.IDataService> saveChangesTo)
        {
            // Write your code here.
            this.ProductFeedbackProperty = new ProductFeedback();
        }

        partial void SliderControl905_Saved()
        {
            // Write your code here.
            this.Close(false);
            Application.Current.ShowDefaultScreen(this.ProductFeedbackProperty);
        }

        partial void SliderControl905_Activated()
        {
            // Write your code here.

        }
    }
}

File: 09/VB/09CustomControls/Client/UserCode/SliderControl905.vb

Namespace LightSwitchApplication

    Public Class SliderControl905

        Private Sub SliderControl905_InitializeDataWorkspace(saveChangesTo As System.Collections.Generic.List(Of Microsoft.LightSwitch.IDataService))
            ' Write your code here.
            Me.ProductFeedbackProperty = New ProductFeedback()
        End Sub

        Private Sub SliderControl905_Saved()
            ' Write your code here.
            Me.Close(False)
            Application.Current.ShowDefaultScreen(Me.ProductFeedbackProperty)
        End Sub

    End Class

End Namespace